readme.md

```diff
@@ -24,26 +24,26 @@ cd path/to/wp-content/themes/<your-theme-name>
 Then, use Composer to install the package.
 
 ```bash
-composer require justintadlock/hybrid-template
+composer require themehybrid/hybrid-template
 ```
 
 ### Register the service provider
 
 You need to register the service provider during your bootstrapping process.  In your bootstrapping code, you should have something like the following:
 
 ```php
-$app = new \Hybrid\Core\Application();
+$theme = new \Hybrid\Core\Application();
```

Comment:
We should promote the use of `$themeslug` here since this is effectively a global (unless it's unset).